app/views/works/index.html.erb

```diff
@@ -0,0 +1,86 @@
+
+
+<h2>List of Works</h2>
+
+<section class="media-table">
+    <h4>Albums</h4>
+    <table class="table">
+      <thead>
+        <tr>
+          <th width=80>Votes</th>
+          <th>Title</th>
+          <th>Created By</th>
+          <th width=100>Published</th>
+          <th width=100>Upvote</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @albums.each do |album|%>
+          <tr>
+            <td><%= album.votes.length %></td>
+            <td><%= link_to "#{album.title}", work_path(album.id) %></td>
+            <td><%= album.creator %></td>
+            <td><%= album.publication_year %></td>
+            <td><%=link_to "Upvote", upvote_path(album.id), method: :post, class: 'btn btn-primary', rel: 'nofollow' %> </a></td>
+          </tr>
+        <%end%>
+      </tbody>
+   </table>
+</section>
+
+<section class="media-table">
+    <h4>Books</h4>
+    <table class="table">
+      <thead>
+        <tr>
+          <th width=80>Votes</th>
+          <th>Title</th>
+          <th>Created By</th>
+          <th width=100>Published</th>
+          <th width=100>Upvote</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <%@books.each do |book|%>
+          <tr>
+           <td><%= book.votes.length %></td>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?